shelfA_marker_id: 100
shelfB_marker_id: 120
